### ISSUE TITLE
Is there a way to increase the page size? [Feature Request] 

### ISSUE THREAD

**Issue by roboyoben** [hundertzehn/mocoapp-api-docs#213]
Hi, thank you for providing the endpoints, they're really helpful!

I was wondering if there's a way to increase the number of entries shown per page? For an organization with tens or hundreds of thousands of entries for some tables, it can take quite a while to load all the data, even if the data is limited to just one year, especially with the rate limiting of 15 requests per 15 seconds.

If not could that be implemented? Perhaps there could be an upper limit to what the page size is allowed to be... something between 1_000 - 10_000 perhaps?

**Comment by agileapplications** [comment]
@roboyoben The reason we have the limit is to make sure other requests are not affected in a negative way. What are you trying to solve? Why do you need to load the data for a full year? Is it a one-time thing our do you do this regularly?

**Comment by roboyoben** [comment]
We want to use the data to analyze time spend on projects and so on by loading it into an analytical tool. That's also why we probably want at least the data for the current year, but for some use cases we might even want it for a greater time frame to get an accurate picture of it. That's something we're still figuring out while developing. This process would run probably once per day, so we were wondering if it's possible to somehow show more entries per request to reduce the number of requests that are needed to fetch the data.

**Comment by agileapplications** [comment]
@roboyoben Thank you for the clarification. Wouldn't it be possible to not query the full years data once per day but just the delta?

**Comment by roboyoben** [comment]
@agileapplications yes that could also be a possible way of doing it. Since we only retrieve the data once per day however we decided that we can run the builds at night for now, so the longer build times are not noticeable. During development of the data model it is still not a good solution, but we can work with this for now. Thank you for your help
